CONTRIBUTING.md

```diff
@@ -0,0 +1,77 @@
+# Guidance on how to contribute
+
+> All contributions to this project will be released under the CC0 public domain
+> dedication. By submitting a pull request or filing a bug, issue, or 
+> feature request, you are agreeing to comply with this waiver of copyright interest.
+> Details can be found in our [TERMS](TERMS.md) and [LICENCE](LICENSE).
+
+
+There are two primary ways to help: 
+ - Using the issue tracker, and 
+ - Changing the code-base.
+
+
+## Using the issue tracker
+
+Use the issue tracker to suggest feature requests, report bugs, and ask questions. 
+This is also a great way to connect with the developers of the project as well
+as others who are interested in this solution.  
+
+Use the issue tracker to find ways to contribute. Find a bug or a feature, mention in
+the issue that you will take on that effort, then follow the _Changing the codebase_ 
+guidance below.
+
+
+## Changing the codebase
```

Comment:
Cool, I'll leave it how it is to avoid overwhelming people. l337 users can use master if they want.